subgraph.template.yaml

```
@@ -21,52 +30,63 @@ dataSources:
         - event: Transfer(indexed address,indexed address,indexed uint256)
           handler: handleTransfer
       file: ./src/erc721/index.ts
+    {{/erc721}}
 
-  - name: ERC7432
+{{#erc1155}}
+  {{#isFirstElement}}
+  - name: ERC1155
+  {{/isFirstElement}} 
+  {{^isFirstElement}}
+  - name: {{name}}
+  {{/isFirstElement}} 
     kind: ethereum
-    network: mumbai
+    network: {{network}}
     source:
-      abi: ERC7432
+      abi: ERC1155
+      address: '{{address}}'
+      startBlock: {{startBlock}}
     mapping:
       kind: ethereum/events
       apiVersion: 0.0.6
       language: wasm/assemblyscript
       entities:
         - Nft
         - Account
-        - RoleAssignment
-        - RoleApproval
       abis:
-        - name: ERC7432
-          file: ./abis/ERC7432.json
+        - name: ERC1155
+          file: ./abis/ERC1155.json
       eventHandlers:
-        - event: RoleGranted(indexed bytes32,indexed address,indexed uint256,address,address,uint64,bool,bytes)
-          handler: handleRoleGranted
-        - event: RoleRevoked(indexed bytes32,indexed address,indexed uint256,address,address)
-          handler: handleRoleRevoked
-        - event: RoleApprovalForAll(indexed address,indexed address,bool)
-          handler: handleRoleApprovalForAll
-      file: ./src/erc7432/index.ts
+        - event: TransferSingle(indexed address,indexed address,indexed address,uint256,uint256)
+          handler: handleTransferSingle
+        - event: TransferBatch(indexed address,indexed address,indexed address,uint256[],uint256[])
+          handler: handleTransferBatch
+      file: ./src/erc1155/index.ts
+{{/erc1155}}
```

Decide if you are gonna leave tabs here. In `erc721` you left
```suggestion
    {{/erc1155}}
```